tests/tck/features/go/SampleLimit.intVid.feature

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2021 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Sample and limit
```

Comment:
How does this work in match statement？Add
 some cases?

Reply:
Don't support.

Reply:
Only supported in go statement.

Reply:
Yes